app/models/manageiq/providers/base_manager/refresh_worker.rb

```diff
@@ -9,7 +9,7 @@ def friendly_name
     @friendly_name ||= begin
       ems = ext_management_system
       if ems.nil?
-        queue_name.titleize
+        queue_name.kind_of?(Array) ? queue_name.collect(&:upcase) : queue_name.titleize
```

Comment:
How about `queue_name.collect(&:titleize).join(", ")` so you'll get: `Ems 1, Ems 2, Ems 3` which is similar to what `queue_name.titleize` would give you for a single ems (`"Ems 1"`).

Reply:
Ah ok, yeah I wasn't too sure what the output should be. So thanks.

Reply:
Lol, I just realised I called `upcase` not `titleize` `#fail`

Reply:
> Lol, I just realised I called upcase not titleize #fail

I think my brain was off, and I just followed Rubocops example! 😂